lib/spack/spack/solver/asp.py

```diff
@@ -878,6 +878,31 @@ def package_compiler_defaults(self, pkg):
                 fn.node_compiler_preference(pkg.name, cspec.name, cspec.version, -i * 100)
             )
 
+    def cfg_required_rules(self, pkg):
```

Comment:
(minor) A name similar to:
```suggestion
    def package_requirement_rules(self, pkg):
```
seems more in line with other methods already present.